iphone/Classes/TiUtils.m

```diff
@@ -1886,7 +1890,7 @@ +(UIImage*)loadBackgroundImage:(id)image forProxy:(TiProxy*)proxy
         }
     }
     else if ([image isKindOfClass:[TiBlob class]]) {
-        resultImage = [image image];
+        resultImage = [(TiBlob*)image image];
```

Comment:
To clarify: This just resolves/cleans a Xcode warning unrelated to the above code.